python/hsfs/feature_group.py

```diff
@@ -562,7 +573,9 @@ def save(
 
         user_version = self._version
         self._feature_group_engine.save(self, feature_dataframe, write_options)
-        if self.statistics_config.enabled:
+        if self.statistics_config.enabled and engine.get_type() == "spark":
```

Comment:
Don't you have the `engine.get_type() == "spark"` also in the `statistics_engine.compute_statistics()`?

Reply:
Yes it's there as well. However the main issue here is that, without that flag, the code would trigger two jobs
- the first to insert data in the feature group
- the second to compute the statistics (which is trigger by the `compute_statistics` method).

The second job is not needed as the statistics are already computed by the first job. That's why I put that flag there.